docs/tutorials/23.2-monorepo-rush.md

```diff
@@ -136,6 +136,75 @@ rush add -p react --dev
 After adding new packages you sometimes need to run `rush update --purge --full` to really update all cache files.
 :::
 
+## Add a Blazor Pilet
+
+Since blazor pilets need some more settings we'll cover them here in more depth. Let's start by adding a `blazorwasm` project. To do so run:
```

Comment:
```suggestion
Since Blazor pilets need some more settings, we'll cover them here in more depth. Let's start by adding a `blazorwasm` project. To do so, run:
```